debian/syncd.install

```diff
@@ -1,3 +1,4 @@
 usr/bin/saidump
 usr/bin/saiplayer
 usr/bin/syncd*
+syncd/usr/bin/* usr/bin
```

Comment:
Now I understand. You put your files into syncd/usr/bin directory.
And make install will not put the files into right places.
I think you need to use bin_SCRIPTS option of automake.
I think it allows you to install your scripts into right places by make install.

P.S.: I think it's better to put your scripts into scripts directory in the root of the project.